src/main/java/com/minecolonies/coremod/client/gui/WindowPostBox.java

```diff
@@ -110,12 +110,17 @@ private void requestClicked(final Button button)
             }
         }
 
+
+        Network.getNetwork().sendToServer(new PostBoxRequestMessage(buildingView, stack.copy(), qty, deliverAvailable));
```

Comment:
seems a bit too much whitespace. You can also delete the uncommented directly. We got git for that =D

Reply:
Yeah, Missed that in my cleanup pass.